### PR TITLE
First pass of Cutscene UI

### PR DESCRIPTION
**Pull request by AndrewJakubowicz**
## Features

 - Tweak to Cinematics to center it correctly according to 9 slice.
 - Cutscene view with vimeo wrapper.
 - Chrome has been pulled out to be more reusable.
 - Center background component made more reusable allowing background image to be passed in.
 - Added base video player to storybook. It has a vimeo and a native mode. (To support china and other non vimeo videos).
 - Updated storybook to serve our static files.

Test "in the wild" with an example by navigating to: http://localhost:3000/cutscene/328690392

Test by looking in storybook with: `npm run storybook`
Here you will see the way to use the video component.

## Todo in future PR:

 - Default video controls shouldn't sit on top of the chrome.
 - subtitle format.
 - Data sources. In china infra fall back to public bucket storage.

### CODE REVIEW

File: ozaria/site/components/cinematic/common/CinematicCanvas.vue [hunk 1]
```diff
@@ -92,8 +92,8 @@ export default {
         || document.documentElement.clientHeight
         || document.body.clientHeight, HEIGHT)
 
-      const width = this.width = Math.min(userWidth, WIDTH)
-      const height = this.height = Math.min(userWidth * CINEMATIC_ASPECT_RATIO, HEIGHT)
+      const height = this.height = Math.min(userWidth * CINEMATIC_ASPECT_RATIO, HEIGHT, userHeight)
+      const width = this.width = Math.min(this.height / CINEMATIC_ASPECT_RATIO)
```
Comment (shubhi1092) [inline ozaria/site/components/cinematic/common/CinematicCanvas.vue]: I think `Math.min` should not be required if this is a single value i.e. `this.height / CINEMATIC_ASPECT_RATIO`

File: ozaria/site/components/cutscene/PageCutscene/index.vue [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+  <layout-chrome>
+    <layout-center-content>
+      <div id="cutscene-player"
+      :style="{ width: width+'px', height: height+'px' }">
+        <base-video
+    videoSrc="https://assets.koudashijie.com/CoCo%E7%AE%80%E4%BB%8B.mp4"
+    :captions="[{
+      label:'English captions',
+      src:'/captions/example.vtt',
+      srclang:'en'
```
Comment (shubhi1092) [inline ozaria/site/components/cutscene/PageCutscene/index.vue]: Is the video source and captions hard-coded for testing purposes? Can we add a comment here describing that?
Also, will there be a cutscenes collection in the database to store this information, and fetch it based on cutscene id/slug?

File: ozaria/site/components/cinematic/common/CinematicCanvas.vue (hunk 1 above)
Comment (AndrewJakubowicz) [reply ozaria/site/components/cinematic/common/CinematicCanvas.vue]: Great pickup. Can remove the `Math.min`.

File: ozaria/site/components/cutscene/PageCutscene/index.vue (hunk 2 above)
Comment (AndrewJakubowicz) [reply ozaria/site/components/cutscene/PageCutscene/index.vue]: Yes. I can add a comment. This is currently hard coded, but will be fetched via a slug from the `cutscene` collection.

**Comment by AndrewJakubowicz** [comment]
@shubhi1092 Thank you so much for the code review. I also don't understand the package-lock.js file. It needs to be updated as I'm adding a new video player library. It always seems to have a very large diff. Will make your recommended changes and merge. Thanks again!

File: ozaria/site/components/cinematic/common/CinematicCanvas.vue (hunk 1 above)
Comment (AndrewJakubowicz) [reply ozaria/site/components/cinematic/common/CinematicCanvas.vue]: This let me simplify the logic in both the cutscene and cinematic. This was a great comment.

**Pull request merged**